src/agent/config/operation.py

```diff
@@ -154,13 +155,18 @@ class LLMSettings(dict):
     A dictionary subclass for the settings of an LLM.
     """
 
-    def __init__(self, name: Union[str, None] = None, api_key: Union[str, None] = None):
+    def __init__(
+        self, name: Union[str, None] = None, api_key: Union[str, None] = None, **kwargs
+    ):
+        # pylint: disable=W0613
```

Comment:
what are we disabling here

Reply:
**kwargs is an unused argument but because of the way i set up llm_opts in in line 279-288, more kwargs were being passed in so i needed to allow that in the this init so it wouldn't give a linting error. its on the next line to keeo it in the scope of that function

Reply:
hmmm are you sure you need kwargs?

Reply:
i think so cuz i get this error when i remove it
TypeError: LLMSettings.__init__() got an unexpected keyword argument 'model'

Reply:
it shouldn't impact functionality since its not used in the function and its optional anyways